x-pack/plugins/canvas/tasks/test.js

```diff
@@ -7,6 +7,7 @@
 import { resolve, join } from 'path';
 
 export default function testTasks(gulp, { mocha }) {
+  require('./mocks/canvas_i18n');
```

Comment:
Also, you might be better off just adding `@kbn/i18n` to the babelhooks setup, and mocking it that way for the tests, instead of doing a runtime check in the code.